Jenkinsfile_CNP

```diff
@@ -98,6 +108,7 @@ withPipeline(type, product, component) {
   }
 
   before('smoketest:preview') {
+    generateRpaContract()
```

Comment:
do we need to generate twice?

Reply:
we will be generating that for both PRs and master, as generation runs schema validation and we want to catch it early. But will be publishing only from the master